helpdesk_mgmt_sla/models/helpdesk_sla.py

```diff
@@ -12,20 +13,49 @@ class HelpdeskSla(models.Model):
 
     name = fields.Char(string="Name", required=True)
     team_ids = fields.Many2many(comodel_name="helpdesk.ticket.team", string="Teams")
-    stage_id = fields.Many2one(comodel_name="helpdesk.ticket.stage", string="Stage")
+    category_ids = fields.Many2many(
+        comodel_name="helpdesk.ticket.category", string="Categories"
+    )
+    tag_ids = fields.Many2many(comodel_name="helpdesk.ticket.tag", string="Tags")
+    stage_ids = fields.Many2many(comodel_name="helpdesk.ticket.stage", string="Stages")
```

Comment:
That this need a migration script stage_id to stage_ids?

Reply:
AFAICS the `stage_id` field wasn't really used in the code before 🤔 I'm not sure what purpose it had